core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -321,6 +321,16 @@ package object config {
       .intConf
       .createWithDefault(3)
 
+  private[spark] val REDUCER_MAX_BLOCKS_IN_FLIGHT_PER_ADDRESS =
+    ConfigBuilder("spark.reducer.maxBlocksInFlightPerAddress")
```

Comment:
I agree this won't resolve all problems, but it is still good to add the limit to prevent a single reducer from fetching too much blocks at a time.